code/game/machinery/camera/tracking.dm

```diff
@@ -49,9 +49,9 @@
 		track.namecounts[name] = 1
 
 		if(ishuman(L))
-			track.humans[name] = L
+			track.humans[name] = WEAKREF(L)
 		else
-			track.others[name] = L
+			track.others[name] = WEAKREF(L)
```

Comment:
This causes runtimes in ai Topics related to tracking where it expects mobs instead of weakrefs.

Reply:
gottcha

Reply:
Should be fixed now